go/pkg/yolosvc/driver_github.go

```diff
@@ -27,13 +34,32 @@ type githubWorker struct {
 	repoConfigs []githubRepoConfig
 }
 
-type githubRepoConfig struct {
-	owner string
-	repo  string
-	// workflows map[int64]struct{}
+func (worker githubWorker) ParseConfig() (bool, error) {
+	if worker.opts.ReposFilter == "" {
+		return true, fmt.Errorf("missing repos filter")
+	}
+
+	var repoConfigs []githubRepoConfig
+
+	for _, repo := range strings.Split(worker.opts.ReposFilter, ",") {
+		parts := strings.Split(repo, "/")
+		if len(parts) != 2 {
+			return false, fmt.Errorf("invalid repo config")
+		}
+		repoConfigs = append(repoConfigs, githubRepoConfig{
+			owner: parts[0],
+			repo:  parts[1],
+		})
+	}
+	if len(repoConfigs) == 0 {
+		return false, fmt.Errorf("invalid repo filter")
+	}
```

Comment:
```suggestion
```

with the current code implementation, this cannot happen.

it will make sense later, if the "range parser" can silently skip invalid repos.